pipelines/broad/reprocessing/exome/ExomeReprocessing.changelog.md

```diff
@@ -1,3 +1,10 @@
+# 2.4.2
+2021-01-26
+
+* Minor changes to support CramToUnmappedBams as an indeependeent versioned pipeline
```

Comment:
Theeree areeee too many eeees heeereeee and on theeee neeeext lineeee

Reply:
Embarrassing keyboard issues :( Should be cleaned up now